### PR TITLE
add note about credentials, npm, pypirc

### PR DESCRIPTION
**Pull request by chriddyp**
@nicolaskruchten - does this help?

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
Just to record my comments: it might not be enough to just tell people to have an NPM/PyPi account... we might need to walk them through how to 'register' their local workstation.

**Comment by chriddyp** [comment]
@nicolaskruchten - how about now?

File: tutorial/plugins.py [hunk 1]
```diff
@@ -347,8 +380,15 @@ class ExampleComponent(dash.development.base_component.Component)
...
+For more information on publishing to PyPI, 
+see this [community blogpost](http://peterdowns.com/posts/first-time-with-pypi.html).
+
 By convention, dash components should adhere to [semver](http://semver.org/).
 Finally, if you'd like, share your component suite with other Dash users in the
 [Dash community forum](https://community.plot.ly/c/dash)!
+
+To be notified of **breaking changes** in the component API, please subscribe to one of these issues:
+- https://github.com/plotly/dash-components-archetype/issues/40
+- https://github.com/plotly/dash-components-archetype/issues/40
```
Comment (nicolaskruchten) [inline tutorial/plugins.py]: these two lines are the same

**Comment by nicolaskruchten** [comment]
Looks great!

**Pull request merged**